tests/test_light_sensor.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import adc
+from greenpithumb import light_sensor
+
+class LightSensorTest(unittest.TestCase):
+
+	def setUp(self):
+		self.mock_adc = mock.Mock(spec=adc.Adc)
+		self.light_sensor = light_sensor.LightSensor(self.mock_adc)
+
+	def test_light_50_pct(self):
+		#light sensor value midway between max and min should 
+		#return 50.0
+		
+		self.mock_adc.read_pin.return_value = (
+			(light_sensor._LIGHT_SENSOR_MAX_VALUE - \
+			light_sensor._LIGHT_SENSOR_MIN_VALUE) / 2) + (
+			light_sensor._LIGHT_SENSOR_MIN_VALUE)
+		light_level = self.light_sensor.get_light_level()
+		self.assertAlmostEqual(light_level, 50.0)
+
+
+	def test_light_out_of_range(self):
+		#ValueError should be raised if light level returned
+		#is either higher than max value or lower than min value

```

Comment:
Comments documenting a function overall should generally be in triple quotes.

See:

https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Docstrings have special utility in unit test functions because the one-line summary in a docstring is what gets printed out when the test fails.